backslash/contrib/slash_plugin.py

```diff
@@ -398,7 +399,7 @@ def _calculate_file_hash(self, filename):
     def _get_git_repo(self, dirname):
         if not os.path.isabs(dirname):
             dirname = os.path.abspath(os.path.join(_PWD, dirname))
-        while dirname != '/':
+        while dirname != _ROOT_DIR:
```

Comment:
@eliadh the handling here should be similar to https://github.com/getslash/slash/pull/918/